prueba.test.js

```diff
@@ -0,0 +1,43 @@
+const { validate, validateStats} = require('./validate');
+const { getLinks, uniqueLinks,stats} = require('./stats');
+const data = '- [Pill de recursión - repositorio](https://github.com/merunga/pildora-recursion)- [Pill de recursión - repositorio](https://github.com/merunga/pildora-recursion)'
+const links =['https://github.com/merunga/pildora-recursion','https://github.com/merunga/pildora-recursion']
+const val ='https://github.com/merunga/pildora-recursion  OK  200'
+const link ='https://github.com/merunga/pildora-recursion'
+describe("getLinks", () =>{
```

Comment:
Recuerda que tu código debe ser sencillo de leer e interpretar.
```suggestion
const link ='https://github.com/merunga/pildora-recursion'

describe("getLinks", () =>{
```